### ISSUE TITLE
Regularization Loss Penalty

### ISSUE THREAD

**Issue by preyasgarg** [victordibia/tpugan#1]
You didn't add the penalty due to the regularization loss while minimizing

**Comment by victordibia** [comment]
Hi @preyasgarg,

Thanks for the note.
Can you provide more information or context? How much improvement might this penalty provide and expected impact on generated images?

Also, a PR would be welcome ..

V

**Comment by preyasgarg** [comment]
I think I wasn't clear on the issue, I meant loss due to Batch Norm. It might hel[ your model or will degrade your model. If you don't want either you can use instance Norm.
I have a few other questions:
Were you able to generate images in the first try or you did multiple tries.
Did you take this code form Tensorflow documentation or made it all by yourself (as I am not able to stabilize my GAN models).
Any suggestions on how to stabilize DCGANs?
I can provide a PR if you still want it, it would be a great pleasure for me?
Thanks
